frontend/src/components/worksheets/items/ItemWrapper.js

```diff
@@ -59,11 +59,15 @@ class ItemWrapper extends React.Component {
             isWorkSheetItem = false;
         }
 
+        const {isDummyItem} = item;
+
         return (
             <div
-                className={classes.container}
+                className={isDummyItem ? "": classes.container}
```

Comment:
what's the standard here - `null`?

Reply:
I don't think there's a standard -- either `null` or `""` work.